web/src/app/services/auth.service.ts

```diff
@@ -12,4 +12,10 @@ export class AuthService {
       new firebase.auth.GithubAuthProvider()
     ));
   }
+
+  signInWithTwitter() {
```

Comment:
Missing return type

Reply:
I think it should be `Observable<UserCredential>`

https://firebase.google.com/docs/reference/js/firebase.auth#usercredential

Reply:
Will check it out. :+1:

Reply:
shall I do this change in the current branch which I am working on, as it may break the task flow.

Reply:
Normally not really, as it will go into `develop`, but for now its fine 👍